File: 1-Iniciante/1019-Conversao_de_tempo-1/1019.py
x=int(input())
segs=0
min=0
hrs=0
for i in range(1,x+1):
    if i%60==0:
        min+=1
        segs=0
    else:
        segs+=1
    if min%60==0 and min!=0:
        hrs+=1
        min=0
print(f"{hrs}:{min}:{segs}")

File: 1-Iniciante/1019-Conversao_de_tempo-1/1019.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        int x = int.Parse(Console.ReadLine());
        int segs=0;
        int min=0;
        int hrs=0;
        for(int i=1; i<x+1;i++){
            if(i%60==0){
                min++;
                segs=0;
            }
            else{
                segs++;
            }
            if(min%60==0 & min !=0){
                hrs++;
                min=0;
            }
        }
        Console.WriteLine($"{hrs}:{min}:{segs}");
    }
}